examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>A model description (Latex and Graphviz) would be great</p><p> </p><pre class="ql-syntax" spellcheck="false">p[0,...,9] ~ Dirichlet(alpha)
y[0,...,9] ~ Multinomial(n, p[0,...,9])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>